### PR TITLE
Exclude OCSP sites for BOD 18-01 compliance calculations

### PR DESCRIPTION
**Pull request by jsf9k**
This pull request:
- Excludes OCSP sites (as listed [here](https://github.com/GSA/data/blob/master/dotgov-websites/ocsp-crl.csv)) from BOD 18-01 compliance calculations
- Still lists OCSP domains in the grand table in the HTTPS report.  Their names are followed by an asterisk to indicate they are special.
- Adds a blurb to the report text that explains this.
- Adds a boolean field to the CSV attachment indicating whether the domain is an OCSP domain or not.

This code has been tested in a full staging deployment on AWS, and it appears to be working.

jsf9k/cyhy-reports#13 is a corresponding pull request that goes along with this one, keeping the CybEx scorecard in sync with the HTTPS reports.

### CODE REVIEW

File: report/https_scan_report.mustache [hunk 1]
```diff
@@ -427,10 +437,10 @@ BOD 18-01 includes \href{https://https.cio.gov/guide/#what-about-domains-that-ar
...
+<<^live_bool>>\color{dhs-blue}\textbf{{<<&domain>><<#ocsp_domain>>*<</oscp_domain>>}} \textit{\ \ Not Live} & \textit{N/A} & \textit{N/A} & \textit{N/A} & \textit{N/A} & \checkmark & <<#hsts_preloaded_bool>> \checkmark <</hsts_preloaded_bool>><<^hsts_preloaded_bool>><<#hsts_preload_pending_bool>> \textit{Pending*} <</hsts_preload_pending_bool>><<^hsts_preload_pending_bool>><<#hsts_preload_ready_bool>> \textit{Preload-ready*} <</hsts_preload_ready_bool>><<^hsts_preload_ready_bool>> \xmark <</hsts_preload_ready_bool>><</hsts_preload_pending_bool>><</hsts_preloaded_bool>> & \textit{No}<</live_bool>>\\
```
Comment (dav3r) [inline report/https_scan_report.mustache]: Typo here:
```<<#ocsp_domain>>``` should be ```<<#oscp_domain>>```
(o**cs**p vs. o**sc**p)

File: report/https_scan_report.mustache [hunk 2]
```diff
@@ -427,10 +437,10 @@ BOD 18-01 includes \href{https://https.cio.gov/guide/#what-about-domains-that-ar
...
...\newline Bad chain}<</https_bad_chain_bool>><<#https_bad_hostname_bool>>\textit{\newline Bad hostname}<</https_bad_hostname_bool>><<#https_expired_cert_bool>>\textit{\newline Expired certificate}<</https_expired_cert_bool>><</domain_supports_https_bool>> & <<#domain_enforces_https_bool>>\checkmark <</domain_enforces_https_bool>><<^domain_enforces_https_bool>>\xmark <</domain_enforces_https_bool>> & <<#domain_uses_strong_hsts_bool>>\checkmark <</domain_uses_strong_hsts_bool>><<^domain_uses_strong_hsts_bool>>\xmark <<#hsts_bool>>\textit{\ \ Low max-age}<</hsts_bool>><</domain_uses_strong_hsts_bool>> & <<^domain_has_weak_crypto_bool>> \checkmark <</domain_has_weak_crypto_bool>><<#domain_has_weak_crypto_bool>> \xmark \textit{\ \ Supports: <<#hosts_with_weak_crypto>><<&weak_crypto_list_str>><</hosts_with_weak_crypto>>} <</domain_has_weak_crypto_bool>> & <<#bod_1801_compliance>>\checkmark <</bod_1801_compliance>><<^bod_1801_compliance>>\xmark <</bod_1801_compliance>> & & <<#domain_has_symantec_cert_bool>> \textit{\textbf{Yes}} <</domain_has_symantec_cert_bool>><<^domain_has_symantec_cert_bool>> \textit{No} <</domain_has_symantec_cert_bool>> \\<</subdomain_scores>><</hsts_preloaded_bool>>
```
Comment (dav3r) [inline report/https_scan_report.mustache]: Same typo here:
```<<#ocsp_domain>>``` should be ```<<#oscp_domain>>```
(o**cs**p vs. o**sc**p)

**Comment by dav3r** [comment]
@jsf9k, this approach makes sense to me.

File: report/https_scan_report.mustache (hunk 1 above)
Comment (jsf9k) [reply report/https_scan_report.mustache]: It's the other way around.  `<</oscp_domain>>` should be `<</ocsp_domain>>`.

Comment (dav3r) [reply report/https_scan_report.mustache]: LOL, that's clearly what I meant ;)

Comment (jsf9k) [reply report/https_scan_report.mustache]: Thanks for pointing it out.  It would have taken me some time to find that.

**Review by dav3r — APPROVED** [review]
Super strong work @jsf9k !  💪 💪 💪

**Pull request merged**